app/src/main/java/com/github/multimatum_team/multimatum/activity/DeadlineDetailsActivity.kt

```diff
@@ -301,10 +318,16 @@ class DeadlineDetailsActivity : AppCompatActivity() {
             val deadline = deadlines[id]!!
             dateTime = deadline.dateTime
             state = deadline.state
+            pdfLink = deadline.pdfPath
             val title = deadline.title
             val description = deadline.description
             val group = deadline.owner
 
+            //display download button if there's a pdf
+            if (pdfLink != "") {
```

Comment:
Please do not represent absent values by empty strings but by null (using `String?`).